app/tests/backend/lib/sow_tab_1.test.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * @jest-environment node
+ */
+import * as XLSX from 'xlsx';
+import {
+  hasDataInRow,
+  readRowData,
+  readData,
+  TAB_ONE_CONSTANTS,
+} from '../../../backend/lib/sow_import/tab_1';
+
+const { COLUMN_KEYS } = TAB_ONE_CONSTANTS;
+
+const buildMockTableData = () => {
+  const rowData = {};
+  const BOOLEAN_KEYS = [
+    TAB_ONE_CONSTANTS.IS_DIRECT_TO_HOME_SATELITE,
+    TAB_ONE_CONSTANTS.IS_IMPACTED_BY_MOBILE_WIRELESS_SERVICE,
+    TAB_ONE_CONSTANTS.IS_INDIGENOUS_COMMUNITY_COLUMN,
+    TAB_ONE_CONSTANTS.IS_WIRED,
+    TAB_ONE_CONSTANTS.IS_WIRELESS,
+  ];
+  COLUMN_KEYS.forEach((key, index) => {
+    if (BOOLEAN_KEYS.includes(key)) {
+      rowData[key] = 'Yes';
+    } else {
+      rowData[key] = `Value_${index}`;
+    }
+  });
+  return rowData;
+};
+
+describe('sow_tab_1 parsing tests', () => {
```

Comment:
maybe we need to check if graphql mutation was called with expected input?
i.e. check https://github.com/bcgov/CONN-CCBC-portal/pull/1604/files#diff-97d71ed34fde88b1575c5db66b79a727bcb0edb0c96b6ff57a4594ad53894774R33

Reply:
Added!